### PR TITLE
misc: make pyright only runs on .py files

### PR DESCRIPTION
**Pull request by superlopuh**
I frequently run `make tests` (thanks again) and this is minor noise when the diff contains files that aren't Python.

### CODE REVIEW

**Review by adutilleul — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -54,9 +54,12 @@ precommit-install:
 precommit:
 	pre-commit run --all
 
+
+
+
```
Comment (AntonLydike) [inline Makefile:59]: Whitespace change? Or was formatting off before?

**Review by AntonLydike — APPROVED** [review]
LGTM

**Review by webmiche — APPROVED** [review]

**Comment by kingiler** [comment]
Do we want `pyright` run on `.ipynb` files?

File: Makefile (hunk 1 above)
Comment (superlopuh) [reply Makefile:59]: ah my b this was the test :)

**Comment by superlopuh** [comment]
That's a good question, I'll add it and see what happens

**Pull request merged**